src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+        size = 0;
+    }
+
+    private static class Node<T> {
+        T item;
+        Node<T> next;
+        Node<T> prev;
+
+        Node(Node<T> prev, T element, Node<T> next) {
+            this.item = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        addLast(value);
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index == size) {
+            addLast(value);
+        } else {
+            Node<T> currentNode = getNode(index);
+            Node<T> prev = getNode(index).prev;
```

Comment:
```suggestion
            Node<T> prev = currentNode.prev;
```
there is no need to call method once again, you execute extra loop